tools/malwarejail/jailme.js

```diff
@@ -1,8 +1,25 @@
-const util = require('node:util');
-require('log-timestamp');
-const vm = require('vm');
-require('./cycle.cjs');
-const minimist = require("minimist")
+import util from 'node:util';

```

Comment:
Generally, how do you know that this files gets executed as ECMA and all the other ones as CommonJS? I'd appreciate if you could point me to where the difference is made.